packages/docs/content/foundation/typography/lists.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: List
+title: Lists
```

Comment:
Since this is being renamed, I think we should consider [adding a redirect](https://github.com/CMSgov/design-system/blob/main/packages/docs/gatsby-node.js#L61) for a while in case people have it bookmarked.